src/SelectBox/SelectBox.stories.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-console */
```

Comment:
What's the point of disabling this ESLint rule on this file?

Reply:
As i said, i copy pasted "Select" component as a base for this component and missed this line hehe.